admin-dev/themes/new-theme/scss/components/_multishop-selector.scss

```diff
@@ -0,0 +1,180 @@
+// stylelint-disable-next-line
+$component-name: shop-selector;
+
+.#{$component-name} {
+  &-search {
+    min-height: auto;
+    padding: 0.3rem 1rem;
+    padding-left: 2.2rem;
+
+    &-container {
+      position: relative;
+      margin-bottom: 1rem;
+
+      @include media-breakpoint-down(sm) {
+        display: none;
+      }
+
+      i {
+        position: absolute;
+        top: 0.5rem;
+        left: 0.75rem;
+        z-index: 1;
+        font-size: 1.2rem;
+        color: $medium-gray;
+        pointer-events: none;
+      }
+
+      .tt-menu {
+        width: 100%;
+        overflow: hidden;
+        background-color: $white;
+        box-shadow: 0 8px 16px 0 rgba(0, 0, 0, 0.1);
+        @include border-bottom-radius(4px);
+
+        .tt-dataset > div {
+          padding: 0.5rem 1rem;
+          cursor: pointer;
+          @include transition(0.25s ease-out);
+
+          &:hover,
+          &.tt-cursor {
+            background-color: #f4fcfd;
```

Comment:
If you can't find an SCSS vars (even inside the UIKit variables which are up because we use the core files of the UIKit), please add one and use it

Reply:
Oh actually the search input was removed from the layout (I copied it from another CSS) so I can clean up all this!!